sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -554,6 +554,12 @@ def _add_argparse_args(cls, parser):
          'worker will install the resulting package before running any custom '
          'code.'))
     parser.add_argument(
+        '--beam_plugins',
+        default=None,
+        help=
+        ('Bootstrap the python process before executing any code by importing '
```

Comment:
Yeah I wasn't sure where this belongs but I think this might be a reasonable place for this for now.